cmd/shell.go

```diff
@@ -302,6 +309,44 @@ func LoadCert(path string) (cert []byte, err error) {
 	return
 }
 
+var ocMu sync.Mutex
+var openConnections int64
+
+// HandlerTimer monitors HTTP performance and sends the details to StatsD.
+func HandlerTimer(handler http.Handler, stats statsd.Statter, prefix string) http.Handler {
```

Comment:
looking out over the amount of state this handlerfunc has in it implies that it might be better off as a struct with a ServeHTTP method on it.